src/js/edu-benefits/components/military-history/MilitaryServiceTour.jsx

```diff
@@ -59,20 +60,20 @@ export default class MilitaryServiceTour extends React.Component {
         <div className="input-section">
           <ErrorableCheckbox
               className="form-field-alert"
-              label="This period of service should be counted towards another education benefit."
-              name="doNotApplyPeriodToSelected"
-              checked={tour.doNotApplyPeriodToSelected}
-              onValueChange={(update) => {onValueChange('doNotApplyPeriodToSelected', update);}}/>
-            {tour.doNotApplyPeriodToSelected
-              ? <ErrorableRadioButtons
-                  errorMessage={validateIfDirty(tour.benefitsToApplyTo, isNotBlank) ? undefined : 'Please provide a response'}
-                  required={tour.doNotApplyPeriodToSelected}
-                  validation={validateIfDirty(tour.benefitsToApplyTo, isNotBlank)}
-                  label="Which benefit should this period of service be applied to?"
-                  name="benefitsToApplyTo"
-                  options={tourBenefits}
-                  value={tour.benefitsToApplyTo}
-                  onValueChange={(update) => {onValueChange('benefitsToApplyTo', update);}}/>
+              label="Apply this service period to the benefit I'm applying for."
+              name="applyPeriodToSelected"
+              checked={tour.applyPeriodToSelected}
+              onValueChange={(update) => {onValueChange('applyPeriodToSelected', update);}}/>
+            {!tour.applyPeriodToSelected
```

Comment:
Do we want this to live under `ExpandableGroup` so we get the nice animations and styles for expanding questions?